doc/content/integrations/storage/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Storage Integration
+description: ""
+summary: Persistent storage for upstream messages.
```

Comment:
add the front matter
`distributions: ["Cloud", "Dedicated Cloud", "Community", "Open Source", "Marketplace Launcher", "Enterprise"]`

and then you can remove the inline distribution shortcodes

Reply:
pr to add this to contributing is still in progress, sorry. see here https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/35/files

also doing `make new integrations/storage` will create a template with an example